Source/Resource/Resources.qrc

```diff
@@ -61,5 +61,89 @@
         <file>Help/Recording/Recording.html</file>
         <file>Help/Recording/record_in_toolbar.jpg</file>
         <file>Help/Recording/record_config.jpg</file>
+        <file alias="Help/qctools.md">../../Docs/qctools.md</file>
```

Comment:
@elderorb was looking to automate this, but for now the resources must be listed here